invenio_accounts/views.py

```diff
@@ -103,3 +112,39 @@ def check_security_settings():
             "SESSION_COOKIE_SECURE setting must be set to True to prevent the "
             "session cookie from being leaked over an insecure channel."
         )
+
+
+@blueprint.route('/security', methods=['GET', 'POST'])
+@login_required
+@register_menu(
+    blueprint, 'settings.security',
+    # NOTE: Menu item text (icon replaced by a user icon).
+    _('%(icon)s Security', icon='<i class="fa fa-user fa-fw"></i>'),
+    order=0)
+@register_breadcrumb(blueprint, 'breadcrumbs.settings.security', _('Security'))
+def security():
+    """View for security page."""
+    sessions = SessionActivity.query.filter_by(user_id=current_user.id).all()
```

Comment:
I would add query methods to SessionActivity since it's likely we need might need them other places:

```SessionActivitiy.query_by_user(current_user.get_id()).all()```

You could also add an instance method:

```session_activity.is_current_session() -> session.id == self.sid_s```. Then it's easy from the template to figure out if a session is the current one.

I think you need to use ``current_user.get_id()`` instead of ``current_user.id`` (i.e. check API in Flask-Login).